pom.xml

```diff
@@ -188,6 +188,7 @@ under the License.
       <version>3.3.0</version>
       <scope>test</scope>
     </dependency>
+    <!-- maven-compat: is must due AbstractMojoTestCase :( -->
```

Comment:
I found that if you don not need to Inject parameters into the Mojo and thus do not test the Mojos run method, one can also go with PlexusTestcase see:
https://github.com/eclipse-tycho/tycho/blob/fdda7227e6cd714c2e120f70dd624cde2cca4ce1/tycho-p2/tycho-p2-facade/src/test/java/org/eclipse/tycho/p2/facade/test/MetadataSerializableImplTest.java

Still, I think this is a bit of a "blind-spot" that there is not a JUnit5 / Non-Compat replacement for a MojoTestCase.